ttls/cli.py

```diff
@@ -113,13 +113,14 @@ async def command_movie(t: Twinkly, args: argparse.Namespace):
 
 async def command_static(t: Twinkly, args: argparse.Namespace):
     await t.interview()
-    m = re.match(r"(\d+),(\d+),(\d+)", args.colour)
+    m = re.match(r"(\d+),(\d+),(\d+),?(\d+)?", args.colour)
```

Comment:
What is the actual pattern here is it 3 or 4 digits? I think I would write this as:

```
    if m := re.match(r"(\d+),(\d+),(\d+)", args.colour):
        …
    elif m := re.match(r"(\d+),(\d+),(\d+),(\d+)", args.colour):
        …
    else
        …
```

Reply:
If possible, please create a separate PR with just the RGBW fixes. Makes feature tracking better.